### ISSUE TITLE
Update tooltip styling for probabilistic chart 

### ISSUE THREAD

**Issue by rachel-labri-tipton** [openclimatefix/nowcasting#384]
## Detailed Description
Looking at the tooltip, the order of information and styles could be better. 

![Screenshot 2023-07-20 at 12 04 29](https://github.com/openclimatefix/nowcasting/assets/86949265/c034afe5-5dd0-4b34-ae9f-2ed6dffff34f)

Comments from @dantravers : 
It might be worth looking a at a few approaches.  
`OCF 10%:`
`OCF Forecast:`
`OCF 90%`
Maybe have the 10% and 90% being slightly smaller font, or the Forecast being in bold, or some way to draw the eye to the forecast?

## Context
Tooltip styling and order of information after Probabilistic Chart updates could have a more logical order to it. 

## Possible Implementation
Work on 2 or 3 versions with different font size and order for the plevel values.

**Comment by rachel-labri-tipton** [comment]
Hello @dantravers and @braddf, 

Here are a few ideas for the tooltip styling. I'm open to feedback but as it's not too high priority, maybe it's just good to have these here for discussion later in August. 

1.  plevel value alignment left with `%` 
a. <img width="205" alt="Screenshot 2023-07-25 at 12 19 50" src="https://github.com/openclimatefix/nowcasting/assets/86949265/710fe07c-9edc-4a66-b4d9-be2c5247a24f">

b. <img width="209" alt="Screenshot 2023-07-25 at 12 35 19" src="https://github.com/openclimatefix/nowcasting/assets/86949265/63e34f18-5f9a-4ce9-b7b0-bd3ffb403a71">

c. <img width="212" alt="Screenshot 2023-07-25 at 12 43 52" src="https://github.com/openclimatefix/nowcasting/assets/86949265/c8d1ded3-5b0c-4af7-8439-a0812bc5ed36">

2. plevel value alignment left w/out `%`. this gives a cleaner look but maybe we don't want to lose the `%`. 

a. <img width="635" alt="Screenshot 2023-07-25 at 12 22 16" src="https://github.com/openclimatefix/nowcasting/assets/86949265/c258850c-3ebe-4c27-8d62-79f87e276491">

b. <img width="221" alt="Screenshot 2023-07-25 at 12 27 23" src="https://github.com/openclimatefix/nowcasting/assets/86949265/460081a6-7918-4890-bfd1-873f95bf018d">

c. <img width="206" alt="Screenshot 2023-07-25 at 12 46 01" src="https://github.com/openclimatefix/nowcasting/assets/86949265/8354e4fd-a3e1-406d-999f-96764aa76edd">

3. for comparison, moving `P10` and `P90` further right. 

a. <img width="215" alt="Screenshot 2023-07-25 at 12 14 14" src="https://github.com/openclimatefix/nowcasting/assets/86949265/23224647-0c04-4a71-b77a-470b43dcf5a1">

b. <img width="216" alt="Screenshot 2023-07-25 at 12 56 11" src="https://github.com/openclimatefix/nowcasting/assets/86949265/68beebca-69a7-474a-8e60-9cc765a8d72b">

**Comment by dantravers** [comment]
Thanks Rachel.  Could we remove the "P" as in the initial issue, and replace with OCF 10%, OCF 90%?  The P isn't incorrect, but it's quite scientific language. 
And I think I like 1a and 1b in terms of styling otherwise.  3 is quite jagged.

**Comment by rachel-labri-tipton** [comment]
Thanks @dantravers, I will replace the P with OCF and have noted 1a and 1b as strong options. 3 is definitely jagged.

**Comment by dantravers** [comment]
Small thing - I just noticed the forecast for 4 hours days 4+ hour forecast.  This is a bit misleading as it's the 4 hour forecast.  I think just remove the + sign, as we don't have it for the normal OCF forecast.  Thanks!

**Comment by braddf** [comment]
@rachel-labri-tipton I agree with 1 a) and b) being the best there - although from a non-stats person and someone who didn't chat through this task - what exactly do the 10% and 90% mean..? 😅 

@dantravers re 4hr view, we actually agreed to change this not _too_ long ago in the tooltips and legend, as when the 4HF goes beyond "now", it is then actually either 4hrs or more than 4hrs, as in each forecast_value is actually `4hr + difference between now and target_time` respectively... I think that's right?

**Comment by dantravers** [comment]
Hi - OK - I think I'm convinced with the 4hr - you are right that it is sometimes more!
